src/server/models/url.ts

```diff
@@ -36,12 +40,14 @@ export const Url = <UrlTypeStatic>sequelize.define(
     longUrl: {
       type: Sequelize.TEXT, // Support >255 chars
       validate: {
-        // Disable long url checks for localstack in development.
-        ...(!DEV_ENV ? { isUrl: true } : {}),
+        urlCheck(url: string) {
+          if (!isValidUrl(url, true)) {
```

Comment:
does this mean production will be able to save the whitelisted localhost URL?

Reply:
Technically, users will not be able to using as we are enforcing https during link creation and edits. But I get the gist here. I will convert the whitelist to a dev-only whitelist. Subsequently, if there is a need for a global whitelist in the future, it should be quite easy to add in.